portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -46,7 +46,11 @@ definition {
 
 		PageTemplatesAdmin.openPageTemplatesAdmin(siteURLKey = "global");
 
-		FormsAdmin.addForm();
```

Comment:
@leticiachiu, can we add a parameter, `groupName` for example, to make this macro support `Guest` and `Global` groups?

This way we'll keep this method faster than previously, what do you think?